standalone/Cargo.toml

```diff
@@ -27,3 +27,4 @@ slog-term = "2.7"
 default = ["dtrace-probes"]
 dtrace-probes = ["propolis/dtrace-probes"]
 crucible = ["propolis/crucible"]
```

Comment:
Now that the crucible backend is using oximeter, shouldn't the feature list `propolis/oximeter` itself here?

Reply:
Updated